web/src/app/add-station-dialogue/add-station-dialogue.component.scss

```diff
@@ -0,0 +1,112 @@
+.dialogue-container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.dialogue-content {
+    background: #fff;
+    width: 40vw;
+    height: 60vh;
```

Comment:
Remove this width and height because it can look weird on some screens it's better to set size based on content.